.github/workflows/release.yaml

```diff
@@ -27,6 +27,7 @@ jobs:
     uses: ./.github/workflows/_build.yaml
     permissions:
       contents: read
+      packages: read
```

Comment:
Could you let me know why we need to have this fix? Is it because action-lint reports this as an error?

Reply:
> Could you let me know why we need to have this fix? Is it because action-lint reports this as an error?

No, that's because the nested `_build_docker.yaml` reusable workflow needs this permission. We need to pass the permissions here too.